style guide (#6259)
circleci/circleci-docs: jekyll/_style/style-headings.adoc

= Headings
:page-layout: classic-docs
:icons: font
:toc: macro
:toc-title:

> * Write headings in a logical sequence. If you read out section titles in order, this should present a good storyline for the doc.

Reviewer: ```suggestion
* Write headings in a logical sequence. If you read section titles in order, the flow should present a good storyline for the doc.
```

> * Do not skip heading levels – try not to jump from, for example, `h2` to `h4`.

Reviewer: ```suggestion
* Do not skip heading levels. For example, try not to jump from `h2` to `h4`, missing out `h3`.
```

* Try not to have a heading with no paragraph text to accompany it. Add at least an overview or intro to what comes next.

* All headings are in sentence case. This means only the first word and proper nouns are capitalized. 
** **Good:** CircleCI docs style guide +
** **Bad:** CircleCI Docs Style Guide

> * Jekyll automatically creates an h1 heading from the doc title, so only use h2 and greater in the body of the doc.

Reviewer: ```suggestion
* Jekyll automatically creates an `h1` heading from the document title, so only use `h2` and greater in the body of the doc.
```
